### PR TITLE
Update with the 1.15.0 release of SPFx

### PR DESCRIPTION
**Pull request by HiroakiOikawa**

### CODE REVIEW

**Comment by shurick81** [comment]
Looks good to me!

**Comment by waldekmastykarz** [comment]
Merged manually. Thank you! 👏

File: README.md [hunk 1]
```diff
@@ -47,9 +47,10 @@ exit
 
 ## Available tags
 
-- **latest**: contains the SharePoint Framework Yeoman generator from the [1.14.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.14?WT.mc_id=m365-0000-wmastyka) release
-- **online**: contains the SharePoint Framework Yeoman generator from the [1.14.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.14?WT.mc_id=m365-0000-wmastyka) release
+- **latest**: contains the SharePoint Framework Yeoman generator from the [1.15.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.15?WT.mc_id=m365-0000-wmastyka) release
+- **online**: contains the SharePoint Framework Yeoman generator from the [1.15.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.15?WT.mc_id=m365-0000-wmastyka) release
 - **onprem**: contains the SharePoint Framework Yeoman generator from the [1.12.1](https://docs.microsoft.com/sharepoint/dev/spfx/release-1.12.1?WT.mc_id=m365-0000-wmastyka) release
+- **1.15.0**: contains the SharePoint Framework Yeoman generator from the [1.15.0](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/release-1.14?WT.mc_id=m365-0000-wmastyka) release
```
[the remark this answers is not in the document]
Comment (HiroakiOikawa) [reply README.md:53]: oh sorry, thanks your follow.

[the remark this answers is not in the document]
Comment (waldekmastykarz) [reply README.md:53]: Minor issue that I've adjusted while merging the PR. Thank you for your help! 👏

**Pull request closed without merging**